### PR TITLE
Add ingress nginx addon for metrics system

### PR DESCRIPTION
**Pull request by dimamo5**
This addon changes the nginx config map to support "heavier" HTTP requests and avoid canceling requests when the proxied server takes more than 5 seconds to connect.

### CODE REVIEW

File: addons/ingress-nginx-metrics/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Deployment
+### AWS
+```
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/ingress-nginx/v1.6.0.yaml
+```
+
+### GCE
+```
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/ingress-nginx/v1.6.0-gce.yaml
+```
```
Comment (JD557) [inline addons/ingress-nginx-metrics/README.md]: Should we change this to point to this repo?

File: addons/ingress-nginx-metrics/v1.6.0.yaml [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+  name: ingress-nginx
+  namespace: kube-ingress
+  labels:
+    k8s-addon: ingress-nginx.addons.k8s.io
+data:
+  # use-proxy-protocol: "true"
+  use-proxy-protocol: "false"
+  client-body-buffer-size: "16k"
+  proxy-connect-timeout: 60
+  proxy-body-size: "10m"
```
Comment (JD557) [inline addons/ingress-nginx-metrics/v1.6.0.yaml:234]: Can you also change the other versions (at least the `v1.6.0-gce`), just in case?

**Review by JD557 — CHANGES_REQUESTED** [review]

File: addons/ingress-nginx-metrics/v1.6.0.yaml (hunk 2 above)
Comment (dimamo5) [reply addons/ingress-nginx-metrics/v1.6.0.yaml:234]: Yes. Changed in 0d0a4f0b5.

File: addons/ingress-nginx-metrics/README.md (hunk 1 above)
Comment (dimamo5) [reply addons/ingress-nginx-metrics/README.md]: Changed in 0d0a4f0b5.

**Review by JD557 — APPROVED** [review]

**Pull request merged**